packages/docs/tutorial/tutorial-requests.yaml

```diff
@@ -303,27 +333,26 @@ _ref:
                         - id: validate
                           type: Validate
                         ################ -------- Copy from here -------- ################
-                        - id: save_data # Make a request to Google Sheets
+                        - id: save_data # Make a request to the SQLite database
                           type: Request
                           params: save_data
                         - id: reset # Reset the form once data has been submitted
                           type: Reset
                         ################ ------- Copy to here ----------- ################
             ```
 
-            #### Step 5.20
+            #### Step 5.2.6
 
-            If you click the submit button, you should see your data submitted to your Google Sheet.
+            If you click the submit button, your form data should be added into the tickets table.
 
             ### What happened
+            We set up the table and column names we will be using in our SQLite database. We need to do this to use the `SQLite` connection.
 
-            We set up the column names we will be using in our Google Sheet. We need to do this to use the `GoogleSheet` connection.
-
-            We defined the `GoogleSheet` connection we will be using in our app, using the credentials we obtained earlier.
+            We defined the `Knex` connection, with the SQLite client.
 
-            We also defined a `GoogleSheetAppendOne` request, to save the data to our sheet, and called that request when clicking the submit button.
+            We also defined a `KnexRaw` request, to save the data to our SQLite database, and called that request when clicking the submit button.
 
```

Comment:
Please also add something about using the button onclick to make the request

Reply:
Change made in 9654eef9a0e0ece57d5e66f6400d8265e9346466